gotham-server/src/server.rs

```diff
@@ -144,6 +145,10 @@ fn get_db(settings: HashMap<String, String>) -> db::DB {
                 None => panic!("Set 'DB = AWS' but 'region' is empty"),
             }
         }
+        "redis" => {
+            let client = redis::Client::open("redis://127.0.0.1/").unwrap();
```

Comment:
Can we inject that as part of a config or an environment variable? Potentially it could be used with a remote url right?